index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OSCD</title>
-    <link rel="stylesheet" href="./hack.css">
+    <link rel="stylesheet" href="./hack_black.css">
```

Comment:
I think there is no need to change the home page styles at all mate (: let it be more like a usual normal website, and change styles only in sprints-related pages.

Reply:
I just think that `hack_clack.css` is cooler (: I'd change it (:

Reply:
Also here https://github.com/oscd-initiative/oscd/pull/4#discussion_r650711910

Reply:
answered here https://github.com/oscd-initiative/oscd/pull/4#discussion_r654148937
let's proceed with the conversation in this thread (:

I believe that the homepage should stay as simple and neutral as possible. Having a new style for each sprint is a good thing. I hope we will see more of them in the upcoming years.

If you strongly believe that it should not be this way — that's fine! Let's vote in the chat (:

Reply:
Guess what, I visited the site again and today I agree (: